File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Categories/CategoryBasic.php
<?php


namespace Ohtap\ECommerce\DomainModels\Objects\Categories;


class CategoryBasic implements ICategoryDetailDetailData
{
    public $ID;
    public $Parent;//CategoryBasic
    public $Photo;//FileData
    public $Title;
    public $Name;
    public $State;
    public $OrderNo;
    public $Description;


}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/Categories/CategoryBasic.cs
using Ohtap.ECommerce.Domain.Models.Objects.Categories;
using Ohtap.ECommerce.Domain.Models.Objects.Files;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.Categories
{
    public class CategoryBasic : ICategoryData
    {
        public string ID { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public int Order { get; set; }
        public string Description { get; set; }
        public int State { get; set; }
        public ICategoryData Parent { get; set; }
        public IFileData Photo { get; set; }
    }
}
